schema/rpc.fbs

```diff
@@ -541,4 +539,16 @@ table SaveFileNotification {
     expected_dir: ComputerDirectory = null;
     /// Recommended filename
     expected_filename: string;
+    /// ID of the SaveFile, needs to be returned by the SaveFileRequest
+    id: uint32;
+}
+
+/// Response of the SaveFileNotification after the user interacts with the file save request
+table SaveFileRequest {
+    /// ID of the SaveFile, given by SaveFileNotification
+    id: uint32;
```

Comment:
Use a newtype struct, also make all primitives optional.

Reply:
what should be wrapped?

Reply:
like `struct SaveFileId`